scripts/build-mailcore2-ios.sh

```diff
@@ -6,13 +6,12 @@ popd > /dev/null
 
 . "$scriptpath/include.sh/build-dep.sh"
 
-url="https://github.com/mailcore/mailcore2.git"
```

Comment:
Could you rather add a parameter to the script to choose a commit or to build master directly?
I'd like to keep a default commit number in that script.